DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -208,6 +212,7 @@ public void UpdateCentreWebsiteDetails_updates_centre()
                     updatedCentre.CentreTelephone.Should().BeEquivalentTo(telephone);
                     updatedCentre.CentreEmail.Should().BeEquivalentTo(email);
                     updatedCentre.CentrePostcode.Should().BeEquivalentTo(postcode);
+                    updatedCentre.ShowCentreOnMap.Should().BeFalse();

```

Comment:
Nice! There are bunch of IDE warnings here, could we try and resolve them as part of this MR?